### ISSUE TITLE
Saved environment variables aren't configured at install time

### ISSUE THREAD

**Issue by insertinterestingnamehere** [conda/conda#2767]
When using saved environment variables as documented at http://conda.pydata.org/docs/using/envs.html#saved-environment-variables, environment variables configured for a given package installed into the currently active environment are not set up properly until the environment is deactivated and reactivated. For example (on Linux):

```
ihenriksen@ubuntu:~$ conda create -n py27 python=2.7
.......(shortened output)
ihenriksen@ubuntu:~$ source activate py27
prepending /home/ihenriksen/miniconda2/envs/py27/bin to PATH
((py27)) ihenriksen@ubuntu:~$ conda install -c insertinterestingnamehere orfeo
.......(shortened output)
((py27)) ihenriksen@ubuntu:~$ echo $OTB_APPLICATION_PATH

((py27)) ihenriksen@ubuntu:~$ source activate root
prepending /home/ihenriksen/miniconda2/bin to PATH
((root)) ihenriksen@ubuntu:~$ source activate py27
prepending /home/ihenriksen/miniconda2/envs/py27/bin to PATH
((py27)) ihenriksen@ubuntu:~$ echo $OTB_APPLICATION_PATH
/home/ihenriksen/miniconda2/envs/py27/lib/otb/applications
```

**Comment by insertinterestingnamehere** [comment]
On a related note, saved environment variables for the root environment aren't available until `root` is explicitly activated. Is that intentional? My intuition is that our addition to peoples' bashrc files should fully activate the root environment, not just manipulate the `PATH`.

**Comment by kalefranz** [comment]
I agree with you. Fixing all this env consistency is one of my big targets for a behavior-changing 5.0 release.

**Comment by kalefranz** [comment]
Fixed with #5044 in conda 4.4!!!
